app/views/users/show.html.erb

```diff
@@ -177,23 +177,19 @@
   </div>
 <% end %>
 
-<div class="home sub-home" id="index-container"
-     data-params="<%= params.merge(user_id: @user.id, class_name: "Article", sort_by: "published_at", sort_direction: "desc").to_json(only: %i[tag user_id q class_name sort_by sort_direction]) %>" data-which="<%= @list_of %>"
-     data-tag=""
-     data-feed="<%= params[:timeframe] || "base-feed" %>"
-     data-articles-since="0">
-  <%= render "users/sidebar" %>
-  <div class="articles-list" id="articles-list">
-    <div class="on-page-nav-controls user-nav-controls" id="on-page-nav-controls">
-      <button class="on-page-nav-butt on-page-nav-butt-left" id="on-page-nav-butt-left" aria-label="nav-button-left">
-        <div class="crayons-icon nav-icon">
-          <%= SiteConfig.left_navbar_svg_icon.html_safe %>
-        </div>
-      </button>
-      <button class="on-page-nav-butt on-page-nav-butt-right" style="margin-top:-1px;" id="on-page-nav-butt-right" aria-label="nav-button-right">
-        <img style="height:28px;width:28px;" src="<%= asset_path "ribbon.svg" %>" alt="right-sidebar-nav">
-      </button>
-    </div>
+<div
+  class="crayons-layout crayons-layout--limited-l crayons-layout--2-cols crayons-layout--2-cols--1-2 pt-4 m:pt-0"
+  id="index-container"
+  data-params="<%= params.merge(user_id: @user.id, class_name: "Article", sort_by: "published_at", sort_direction: "desc").to_json(only: %i[tag user_id q class_name sort_by sort_direction]) %>" data-which="<%= @list_of %>"
```

Comment:
wonder if we can move the value of `data-params` up to the controller 👁️

Reply:
Since I don't know the answer, it's probably not something you want me to do :D